src/user-guide/task-implementation/run-modes.rst

```diff
@@ -0,0 +1,211 @@
+.. _task-run-modes:
+
+Task Run Modes
+==============
+
+.. versionadded:: 8.4.0
+
+   * Before 8.4.0 run modes could only be set for the workflow, not
+     for individual tasks.
+   * Skip mode.
+
+
+As well as the default task implementation ("live" mode)
+other implementations can be selected:
+
+* By setting the scheduler run mode as an argument to the
+  ``cylc play --mode`` command ("simulation" and "dummy" modes).
+* By setting :cylc:conf:`[runtime][<namespace>]run mode` for
+  "skip" mode.
+
+#. :ref:`task-run-modes.simulation`: Intended to be used
+   when developing a workflow, the scheduler simulates
+   running a task without creating a "real" job.
+#. :ref:`task-run-modes.dummy`: Intended to be used
+   when developing a workflow, the scheduler replaces
+   the workflow scripts with dummy scripts and returns
+   to the default ("live") submission pathway.
+#. :ref:`task-run-modes.skip`: Intended to allow more control of the workflow,
+   the scheduler marks task outputs as completed.
+
+.. note::
+
+   The simulation and dummy workflow run modes ignore the
+   :cylc:conf:`[runtime][<namespace>]run mode` setting.
+
+
+.. _task-run-modes.dummy:
+
+Dummy Mode
+----------
+
+**Dummy mode** replaces real jobs with background jobs on the
+scheduler host which use ``sleep`` to simulate the run length according to
+the settings described for simulation mode.
+This avoids :term:`job runner` directives that request compute
+resources for real workflow tasks, and it allows any workflow configuration to
+be run locally in dummy mode.
```

Comment:
Partially redundant - should be combined with the description higher up?

Reply:
Deliberately somewhat similar - the other text is a top level abstract, this is the introduction to the documentation for dummy mode.

Reply:
Resolved: Fell out of use when I moved the workflow run modes back to the other file